src/shims/ApiCompatBaseline.netcoreapp.netstandard.txt

```diff
@@ -1,46 +1,29 @@
 Compat issues with assembly mscorlib:
-TypesMustExist : Type 'System.TupleExtensions' does not exist in the implementation but it does exist in the contract.
-TypesMustExist : Type 'System.Runtime.CompilerServices.TupleElementNamesAttribute' does not exist in the implementation but it does exist in the contract.
-Compat issues with assembly netstandard:
 TypeCannotChangeClassification : Type 'System.RuntimeArgumentHandle' is a 'ref struct' in the implementation but is a 'struct' in the contract.
 TypeCannotChangeClassification : Type 'System.TypedReference' is a 'ref struct' in the implementation but is a 'struct' in the contract.
-Compat issues with assembly System.Core:
-TypesMustExist : Type 'System.Security.Cryptography.ECCurve' does not exist in the implementation but it does exist in the contract.
-TypesMustExist : Type 'System.Security.Cryptography.ECParameters' does not exist in the implementation but it does exist in the contract.
-TypesMustExist : Type 'System.Security.Cryptography.ECPoint' does not exist in the implementation but it does exist in the contract.
-
-# Compat issues complaining about class vs delegate and class vs struct are because of a bug in APICompat tool where the implementation is picking
-# the wrong core assembly. It is picking System.Runtime instead of System.Private.CoreLib, there isn't any straight forward way to fix so baselining.
-
+Compat issues with assembly netstandard:
 TypeCannotChangeClassification : Type 'System.RuntimeArgumentHandle' is a 'ref struct' in the implementation but is a 'struct' in the contract.
```

Comment:
That doesn't look right. In `netstandard` it's also marked as `ref struct`

Reply:
Is that new in `ns21`? This check also appears in the old baseline: https://github.com/dotnet/corefx/blob/master/src/shims/ApiCompatBaseline.netcoreapp.netstandard20.txt#L5

Reply:
It looks like Immo fixed this in 2.1: https://github.com/dotnet/standard/commit/31e10d1032184df098b40331a0bb0276acfc294f

As a result we should expect that these are no longer needed for APICompat runs between 2.1 and netcoreapp.

Reply:
Yeah I just found that myself. Looking to see if these magically disappear on a netcoreapp build, otherwise I'll dig further.

Reply:
Ah, it's because we only updated the other check (`NetStandardOnly`) to run against ns2.1, while this one still runs against ns20: https://github.com/dotnet/corefx/pull/34796/files#diff-085f6f9ba6ffb73170783b85bbf23af8R78